### DISCUSSION TITLE
Account deletion

### DISCUSSION THREAD

**Discussion by nextdimension** [deepgram/community#637] [General help] [answered]
Please allow me to delete my account. Or delete it for me.

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jpvajda** [comment]
@nextdimension  Let me see if I can help you with that. Can you provide me the `project ID(s)` for your account?

**Reply by nextdimension** [reply]
not in public.

**Reply by nextdimension** [reply]
actually, it has no personal information tied so its ok.
f030ab43-11c6-4ca6-beb1-dd2149a6802c

**Reply by nextdimension** [reply]
Can you delete my email and all my data and account

**Reply by nextdimension** [reply]
You guys really need a way for us to self service this. Massive oversight imo

**Comment by jpvajda** [comment]
@nextdimension  We understand it can be frustrating to not have an easy way to delete an account without asking for help, it's definitely something on our product roadmap to give people more control over being able to do exactly this. Thanks for being patient.

**Comment by jpvajda** [comment, marked as answer]
@nextdimension  just one more step, I'll just need to validate which email account you've used with Deepgram so if you can send us an email to `devrel@deepgram.com` from the account you used to sign-up -  I can take that final validation step.

**Comment by jpvajda** [comment]
This has been resolved via email.
